web/annotation_layer_builder.js

```diff
@@ -128,24 +135,13 @@ class AnnotationLayerBuilder {
 
     // Create an annotation layer div and render the annotations
     // if there is at least one annotation.
-    const div = (this.div = document.createElement("div"));
-    div.className = "annotationLayer";
-    this.#onAppend?.(div);
```

Comment:
Why is this removed here, since that seems unrelated!?
Please move this back.

Reply:
Done. Also I've set `this.div.hidden` to `false` at the end of `injectLinkAnnotations` as a result, let me know if you prefer a `show` method.